README.md

```diff
@@ -69,15 +74,17 @@ In each JSON we provide information about the bounding boxes detected in each in
 This allows us to evaluate the performance of an object detection model based on the ground truth and true positive 
 bounding boxes.
 
-## Database
+#### Database
 
-To access the database is necessary to create a `database.json` file inside the `database` folder.
+To access a database in Azure Machine Learning it is necessary to create a `database.json` file inside the `database` folder.
 An example of the structure can be found in the folder under the name `database.example.json`.
 
-The file contains the following information:
+This database.json file should include the following information:
 ``` 
-    hostname:       hostname address of the database
-    username:       managed identity name in production
-    database_name
-    client_id:      client id of the managed identity
+    client_id:      client id of the managed identity in Azure
 ```
+
+#### Hyperparamters for panoramic images
+TODO
```

Comment:
What are the hyp param changes that we did to YOLO to make it optimally work with Panos?